src/metabase/channel/impl/email.clj

```diff
@@ -307,7 +267,9 @@
                                                                           (urls/notification-management-url)
                                                                           (pulse-unsubscribe-url-for-non-user (:id dashboard_subscription) non-user-email))
                                                     :filters           (when (seq parameters)
-                                                                         (render-filters parameters))})
+                                                                         (some->> parameters
+                                                                                  (impl.util/remove-inline-parameters dashboard_parts)
+                                                                                  (impl.util/render-filters)))})
```

Comment:
nit: Can subsume the `when` into the `some->>` by starting the threading chain with `(seq parameters)`